stl/inc/stacktrace

```diff
@@ -0,0 +1,347 @@
+// stacktrace standard header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _STACKSTRACE_
+#define _STACKSTRACE_

```

Comment:
Typo: `_STACKSTRACE_` should be `_STACKTRACE_`. Occurs in the `#endif` comment too.